### PR TITLE
[Console] [WCM] Add documentation about the AsLockedCommand attribute

### PR DESCRIPTION
**Pull request by NeilPeyssard**
See https://github.com/symfony/symfony/pull/53995

### CODE REVIEW

File: console/lockable_command.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+=======================================================
+
+You can use `locks`_ to prevent the same command from running multiple times on
+the same server. The :doc:`Lock component </components/lock>` provides multiple
+classes to create locks based on the filesystem (:ref:`FlockStore <lock-store-flock>`),
+shared memory (:ref:`SemaphoreStore <lock-store-semaphore>`) and even databases
+and Redis servers.
+
+Using the AsLockedCommand attribute
+-----------------------------------
```
Comment (MrYamous) [inline console/lockable_command.rst]:
```suggestion
Using the ``#[AsLockedCommand]`` attribute
------------------------------------------
```

File: console/lockable_command.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+The Console component provides an attribute called ``AsLockedCommand`` that automatically
```
Comment (MrYamous) [inline console/lockable_command.rst]:
```suggestion
The Console component provides an attribute ``#[AsLockedCommand]`` that automatically
```

File: console/lockable_command.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        {
+            $output->writeln('Purging the database...');
+
+            return Command::SUCCESS;
+        }
+    }
+
+.. versionadded:: 7.1
+
+    The ``AsLockedCommand`` attribute was introduced in Symfony 7.1.
```
Comment (MrYamous) [inline console/lockable_command.rst]:
```suggestion
    The ``#[AsLockedCommand]`` attribute was introduced in Symfony 7.1.
```

File: console/lockable_command.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        public static function getLockKey(InputInterface $input): string
+        {
+            return $input->getArgument('lock-key');
+        }
+    }
+
+Using the LockableTrait
+-----------------------
+
+In addition, the Console component provides a PHP trait called ``LockableTrait``
```
Comment (MrYamous) [inline console/lockable_command.rst]:
```suggestion
In addition, the Console component provides a :class:`Symfony\\Component\\Console\\Command\\LockableTrait`
```

:class:`Symfony\\Component\\Console\\Command\\LockableTrait`

File: console/lockable_command.rst (hunk 1 above)
Comment (NeilPeyssard) [reply console/lockable_command.rst]: Done :+1:

File: console/lockable_command.rst (hunk 2 above)
Comment (NeilPeyssard) [reply console/lockable_command.rst]: Done :+1:

File: console/lockable_command.rst (hunk 4 above)
Comment (NeilPeyssard) [reply console/lockable_command.rst]: Done :+1:

File: console/lockable_command.rst (hunk 3 above)
Comment (NeilPeyssard) [reply console/lockable_command.rst]: Done :+1:

**Comment by javiereguiluz** [comment]
Closing because the related code PR was closed without merging: https://github.com/symfony/symfony/pull/53995

**Pull request closed without merging**